app/Console/Commands/InstallCommand.php

```diff
@@ -0,0 +1,106 @@
+<?php
+
+namespace App\Console\Commands;
+
+use Illuminate\Console\Command;
+
+class InstallCommand extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'council:install';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Simplify installation process';
+
+    /**
+     * Execute the console command.
+     *
+     * @return mixed
+     */
+    public function handle()
+    {
+        $this->intro();
+
+        if (!file_exists('.env')) {
+            exec('mv .env.example .env');
+            $this->line("\r\n.env file successfully created\r\n");
+        }
+        
+        exec('composer install');
+        $this->line("\r\n ~ App dependencies successfully installed\r\n");
+
+        exec('php artisan key:generate');
```

Comment:
Why don't we directly call `Artisan` here?

Reply:
That would probably be better, yes